thunder/__init__.py

```diff
@@ -563,25 +563,38 @@ def get_computation_and_inputs(*args, **kwargs):
                     # thunder_backward may recursively call compile and wraps the result in a
                     # torch.autograd.Function to support embedding of Thunder-compiled
                     # functions in torch's Autograd
+
+                    # Currently split_forward_backward also includes
+                    # transform_for_execution and various sorting of symbols,
+                    # applying transform_for_execution after this would be
+                    # breaking the order of operations
                     computation_trc, backward_trc = split_forward_backward(computation_trc, cd, cs, *inps)
                     # Note computation_trc and backward_trc have been appended to cs.last_(backward_)traces
                     # by split_forward_backward
-
-            cs.last_computation_transformation_start = time.time_ns()
-
-            ## EPILOGUE and TRANSFORMS should not mix...
-            # applies transforms
-            for transform in additional_transforms:
-                computation_trc = transform(computation_trc, executors_list=cd.executors_list)
-                computation_traces.append(computation_trc)
-
-            with langctxs.langctx(cd.langctx):
-                extraces = transform_for_execution(
-                    computation_trc,
-                    executors_list=cd.executors_list,
-                )
-            extrace = extraces[-1]
-            comp = extrace.python_callable()
+                    extraces = []
+                    check(
+                        not additional_transforms,
```

Comment:
Ah, so, my suspicion was not something I have imagined while being too tired? :)

Reply:
I always confuse what do I need to pass to `check`.